src/control/server/ctl_storage_rpc_test.go

```diff
@@ -276,2055 +648,81 @@ func TestServer_CtlSvc_StorageScan_PreEngineStart(t *testing.T) {
-//					ScanRes: &storage.BdevScanResponse{Controllers: *tc.csCtrlrs},
-//				}
-//			}
-//
-//			var engineCfgs []*engine.Config
-//			for i, sc := range tc.storageCfgs {
-//				log.Debugf("storage cfg contains bdevs %v for engine %d", sc.Bdevs(), i)
-//				engineCfgs = append(engineCfgs,
-//					engine.MockConfig().
-//						WithStorage(sc...).
-//						WithTargetCount(tc.engineTargetCount[i]))
-//			}
-//			sCfg := config.DefaultServer().WithEngines(engineCfgs...)
-//			cs := mockControlService(t, log, sCfg, csbmbc, tc.smbc, tc.smsc)
-//
-//			for idx, ec := range engineCfgs {
-//				var ebmbc *bdev.MockBackendConfig
-//				if tc.eCtrlrs != nil && len(tc.eCtrlrs) > idx {
-//					log.Debugf("bdevs %v to be returned for engine %d scan",
-//						*tc.eCtrlrs[idx], idx)
-//					ebmbc = &bdev.MockBackendConfig{
-//						ScanRes: &storage.BdevScanResponse{
-//							Controllers: *tc.eCtrlrs[idx],
-//						},
-//					}
-//				}
-//
-//				// replace harness instance with mock I/O Engine
-//				// to enable mocking of harness instance drpc channel
-//				sp := storage.MockProvider(log, idx, &ec.Storage,
-//					cs.storage.Sys, // share system provider cfo
-//					scm.NewMockProvider(log, tc.smbc, nil),
-//					bdev.NewMockProvider(log, ebmbc), nil)
-//				te := newTestEngine(log, false, sp, ec)
-//
-//				if tc.enginesNotReady {
-//					te.ready.SetFalse()
-//				}
-//
-//				// mock drpc responses
-//				dcc := new(mockDrpcClientConfig)
-//				if tc.junkResp {
-//					dcc.setSendMsgResponse(drpc.Status_SUCCESS,
-//						makeBadBytes(42), nil)
-//				} else if len(tc.drpcResps) > idx {
-//					t.Logf("setting %d drpc responses for engine %d",
-//						len(tc.drpcResps[idx]), idx)
-//					dcc.setSendMsgResponseList(t, tc.drpcResps[idx]...)
-//				} else {
-//					t.Fatal("drpc response mocks unpopulated")
-//				}
-//				te.setDrpcClient(newMockDrpcClient(dcc))
-//				te._superblock.Rank = ranklist.NewRankPtr(uint32(idx + 1))
-//				for _, tc := range te.storage.GetBdevConfigs() {
-//					tc.Bdev.DeviceRoles.OptionBits = storage.OptionBits(storage.BdevRoleAll)
-//				}
-//				md := te.storage.GetControlMetadata()
-//				md.Path = "/foo"
-//				md.DevicePath = md.Path
-//
-//				cs.harness.instances[idx] = te
-//			}
-//			cs.harness.started.SetTrue()
-//
-//			if tc.req == nil {
-//				tc.req = &ctlpb.StorageScanReq{
-//					Scm:  new(ctlpb.ScanScmReq),
-//					Nvme: new(ctlpb.ScanNvmeReq),
-//				}
-//			}
-//
-//			if tc.scanTwice {
-//				_, err := cs.StorageScan(test.Context(t), tc.req)
-//				test.CmpErr(t, tc.expErr, err)
-//				if err != nil {
-//					return
-//				}
-//			}
-//
-//			resp, err := cs.StorageScan(test.Context(t), tc.req)
-//			test.CmpErr(t, tc.expErr, err)
-//			if err != nil {
-//				return
-//			}
-//
-//			if diff := cmp.Diff(tc.expResp, resp, defStorageScanCmpOpts...); diff != "" {
-//				t.Fatalf("unexpected response (-want, +got):\n%s\n", diff)
-//			}
-//		})
-//	}
-//}
-
-func TestServer_checkTmpfsMem(t *testing.T) {
```

Comment:
This wasn't meant to be removed, was it?

Reply:
nope, good catch, restored in the next commit, thanks